flink-kubernetes-operator-autoscaler/src/main/java/org/apache/flink/kubernetes/operator/autoscaler/ScalingMetricEvaluator.java

```diff
@@ -66,12 +67,15 @@ public Map<JobVertexID, Map<ScalingMetric, EvaluatedScalingMetric>> evaluate(
         var scalingOutput = new HashMap<JobVertexID, Map<ScalingMetric, EvaluatedScalingMetric>>();
         var metricsHistory = collectedMetrics.getMetricHistory();
         var topology = collectedMetrics.getJobTopology();
+        var excludeVertexIdList = Arrays.asList(conf.get(AutoScalerOptions.EXCLUDE_VERTEX_IDS).split(","));
```

Comment:
No need to do this here if we use a list type like other options, e.g.  see `classloader.parent-first-patterns.default`.

Reply:
ok, thanks, will do it